pages/redirect.vue

```diff
@@ -0,0 +1,44 @@
+<!--
+ * @Description:
+ * @Author: Floyd Li (floyd.li@outlook.com)
+ * @Date: 2023-03-24 16:10:46
+ * @LastEditors: Floyd Li (floyd.li@outlook.com)
+ * @LastEditTime: 2023-03-25 17:49:29
+-->
```

Comment:
```suggestion
```

Reply:
hmmm this comment is generated by editor plugin. gonna disable it for this project